include/openssl/thread.h

```diff
@@ -116,7 +116,7 @@ OPENSSL_EXPORT int AWSLC_thread_local_clear(void);
 // calls to |AWSLC_thread_local_clear|.
 OPENSSL_EXPORT int AWSLC_thread_local_shutdown(void);
 
-// Deprecated functions.
+// General No-op Functions [Deprecated].
```

Comment:
Can we add OPENSSL_DEPRECATED to all the functions below?

Reply:
I had been pondering this since we enable thread safety by default, so it was OK for these functions to do nothing and not alert the consuming application.
I dug into[ the OpenSSL headers](https://github.com/openssl/openssl/blob/4514e02cdfc96589d5e8ab0a08942fafa8e418ae/include/openssl/crypto.h.in#L297) and noticed there are some deprecated functions though, we can mark those to align with upstream.